### ISSUE TITLE
Document mixin parameter support in number-leading-zero

### ISSUE THREAD

**Issue by jackrussell1996** [stylelint/stylelint#3675]
> Clearly describe the bug

When providing parameters for Less mixins I do not get the number-leading-zero warning when using a number without the leading zero.

> Which rule, if any, is the bug related to?

`number-leading-zero`

> What CSS is needed to reproduce the bug?

e.g.

```less
.border-radius(@radius) {
  -webkit-border-radius: @radius;
  -moz-border-radius: @radius;
  border-radius: @radius;
}

#mydiv {
  .border-radius(.5px);
}
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "number-leading-zero": "always"
  }
}
```

> Which version of stylelint are you using?

`9.2.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

"CLI with `stylelint header.less`"

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

"Yes, it's related to Less mixins."

> What did you expect to happen?

"Warning for number-leading-zero."

> What actually happened (e.g. what warnings or errors did you get)?

"No warning for number-leading-zero:"

**Comment by jeddy3** [comment]
@jackrussell1996 Thanks for the report and for using the template.

The built-in rules focus on standard CSS constructs e.g. the numbers within declaration and media feature values. You can create a 
[plugin](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md), using the built-in `number-leading-zero` rule as blueprint, to check the numbers within the parameters of Less mixins.

I'll flag this as a documentation issue, though.
